Can't build version 1.2.0 nor master

Steps to reproduce:

``` bash
git clone https://github.com/ArthurSonzogni/json-tui
cd json-tui
cmake -B build
```

output:
```
-- Populating ftxui
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/json-tui/build/_deps/ftxui-subbuild
[ 12%] Creating directories for 'ftxui-populate'
[ 25%] Performing download step (git clone) for 'ftxui-populate'
Cloning into 'ftxui-src'...
remote: Enumerating objects: 1518, done.        
remote: Counting objects: 100% (1518/1518), done.        
remote: Compressing objects: 100% (700/700), done.        
remote: Total 1518 (delta 804), reused 1298 (delta 718), pack-reused 0        
Receiving objects: 100% (1518/1518), 8.03 MiB | 7.47 MiB/s, done.
Resolving deltas: 100% (804/804), done.
fatal: reference is not a tree: 3dc215e6c0eecb1607d428af0295bcd98db74b8f
CMake Error at ftxui-subbuild/ftxui-populate-prefix/tmp/ftxui-populate-gitclone.cmake:40 (message):
  Failed to checkout tag: '3dc215e6c0eecb1607d428af0295bcd98db74b8f'


make[2]: *** [CMakeFiles/ftxui-populate.dir/build.make:98: ftxui-populate-prefix/src/ftxui-populate-stamp/ftxui-populate-download] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/ftxui-populate.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
CMake Error at /usr/share/cmake/Modules/FetchContent.cmake:1087 (message):
  Build step for ftxui failed: 2
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FetchContent.cmake:1216:EVAL:2 (__FetchContent_directPopulate)
  /usr/share/cmake/Modules/FetchContent.cmake:1216 (cmake_language)
  CMakeLists.txt:33 (FetchContent_Populate)


-- Configuring incomplete, errors occurred!
```

Version 1.1.0 builds fine, though.

Thanks in advance :)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,7 +11,7 @@ FetchContent_Declare(ftxui
   GIT_REPOSITORY https://github.com/ArthurSonzogni/ftxui
   GIT_TAG 3dc215e6c0eecb1607d428af0295bcd98db74b8f
   GIT_PROGRESS TRUE
-  GIT_SHALLOW TRUE
+  GIT_SHALLOW FALSE
 )
 
 FetchContent_Declare(json
@@ -25,7 +25,7 @@ FetchContent_Declare(args
   GIT_REPOSITORY https://github.com/Taywee/args
   GIT_TAG a48e1f880813b367d2354963a58dedbf2b708584
   GIT_PROGRESS TRUE
-  GIT_SHALLOW TRUE
+  GIT_SHALLOW FALSE
 )
 
 FetchContent_GetProperties(ftxui)

